.github/workflows/quic.yml

```diff
@@ -158,11 +158,12 @@ jobs:
             tls: 'openssl'
           - os: 'ubuntu-22.04'
             tls: 'openssl3'
+            xdp: '-UseXdp'
     uses: microsoft/msquic/.github/workflows/build-reuse-unix.yml@main
     with:
       os: ${{ matrix.os }}
       tls: ${{ matrix.tls }}
-      xdp: ${{ matrix.io == 'xdp' && '-UseXdp' || '' }}
+      xdp: ${{ matrix.xdp }}
```

Comment:
The `Download Artifacts` step below will need to be updated for linux xdp too, right?